packages/ag-charts-enterprise/src/box-plot/boxPlotSeries.ts

```diff
@@ -79,11 +104,18 @@ export class BoxPlotSeries extends CartesianSeries<
     strokeOpacity = 1;
 
     @Validate(OPT_LINE_DASH)
-    lineDash?: number[] = [0];
+    lineDash: number[] = [0];
 
     @Validate(NUMBER(0))
     lineDashOffset: number = 0;
 
+    @Validate(OPT_FUNCTION)
+    formatter?: (params: AgBoxPlotSeriesFormatterParams<BoxPlotNodeDatum>) => AgBoxPlotSeriesFormat = undefined;
+
+    cap = new BoxPlotSeriesCap();
+
+    whisker = new BoxPlotSeriesWhisker();
+
     tooltip = new SeriesTooltip<AgCartesianSeriesTooltipRendererParams>();
```

Comment:
Nit: Just wondering, could/should these properties be `readonly` if the reference never changes?

Reply:
np, I was merely following existing code examples
so shouldn't this also be the case for `tooltip`

Reply:
Yes, SGTM for `tooltip` :-)